lib/crypto.js

```diff
@@ -663,10 +663,10 @@ function filterDuplicates(names) {
 
 // Legacy API
 exports.__defineGetter__('createCredentials',
-  internalUtil.deprecate(function() {
-    return require('tls').createSecureContext;
-  }, 'crypto.createCredentials is deprecated. ' +
-     'Use tls.createSecureContext instead.'));
+                         internalUtil.deprecate(function() {
+                           return require('tls').createSecureContext;
+                         }, 'crypto.createCredentials is deprecated. ' +
+                           'Use tls.createSecureContext instead.'));
```

Comment:
I'm not sure. It looks wrong to me because both strings are not aligned to each other